docs/blog/introducing-weps.rst

```diff
@@ -0,0 +1,81 @@
+Introducing Write the Docs Enhancement Proposals
+================================================
+
+Write the Docs has been growing each year for the past 7,
+and the scope of the community has expanded dramatically.
+We started with a single conference,
+and now we have multiple conferences,
+over 50 meetups around the world,
+multiple online spaces,
+and many other ongoing projects.
```

Comment:
```suggestion
Write the Docs began seven years ago with a single conference.  
It has now grown to multiple conferences worldwide, over 50 international meetups and many other projects. 
```

Reply:
Edit as needed :-p

Reply:
I didn't quite understand this either. I think it was trying to replace the entire paragraph. I've taken that copy and applied it in a commit.